opam

```diff
@@ -1,17 +0,0 @@
-opam-version: "1"
```

Comment:
The `opam` file should _not _ be delete, it should be renamed `xapi-networkd.opam`.
You need to delete the `remove: ...` section
You need to replace the `build: ...` section with
```
build: ["jbuilder" "build" "-p" name "-j" jobs]
build-test: ["jbuilder" "runtest" "-p" name]
```

You also need to remove `bisect_ppx` and `ocamlfind` from the `depends:` and add `"jbuilder" {build & >= "1.0+beta11"}` and `"xapi-stdext" {>= 3.0.0}"